kadenza/calibration.py

```diff
@@ -86,6 +86,19 @@ def smear_frame(self, channel):
             frame[:, columns[idx]] = smear_values[idx]
         return frame
 
+    def get_smear_at_columns(self, columns, channel):
+        """
+        colnums : list-like of integers
+        """
```

Comment:
My only comment is that the docs should say that the return values are in units ADU